src/sentry/web/api.py

```diff
@@ -750,18 +765,21 @@ def post(self, request, project, **kwargs):
             if has_event_attachments:
                 attachments.append(CachedAttachment.from_upload(file))
 
-        try:
-            state = process_minidump(minidump)
-            merge_process_state_event(data, state)
-        except ProcessMinidumpError as e:
-            minidumps_logger.exception(e)
-            track_outcome(
-                project.organization_id,
-                project.id,
-                None,
-                Outcome.INVALID,
-                "process_minidump")
-            raise APIError(e.message.split('\n', 1)[0])
+        if _minidump_refactor_enabled(project.id):
```

Comment:
Can we add a metric or two here (i.e. this decision, and timing for `process_minidump` below) to have some visibility?

Reply:
done